src/main/java/io/confluent/kafkarest/KafkaRestConfig.java

```diff
@@ -57,6 +57,18 @@
       + "the connection string as hostname1:port1,hostname2:port2,hostname3:port3/chroot/path.";
   public static final String ZOOKEEPER_CONNECT_DEFAULT = "localhost:2181";
 
+  public static final String BOOTSTRAP_SERVERS_CONFIG = "bootstrap.servers";
+  private static final String
+      BOOTSTRAP_SERVERS_DOC =
```

Comment:
Might be worth calling out that the only protocol currently supported is PLAINTEXT, hence why `bootstrap.servers` doesn't take a protocol (eg `PLAINTEXT://localhost:9092`).